src/main/java/com/edhou/songr/services/FileUploadService.java

```diff
@@ -0,0 +1,27 @@
+package com.edhou.songr.services;
+
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+
+@Service
```

Comment:
The annotation @Service tells SpringMVC to use this class when I ask for a FileUploadService using @Autowire